src/DotvvmAcademy.Web/Course/010_concepts/en/20_view.md

```diff
@@ -11,21 +11,26 @@ CodeTask:
 
 # The View
 
-__Views__ are everything users see and interact with. ViewModels provide the data and logic for them. In DotVVM, they are written in __Dothtml__, a flavour of _HTML_.
+__Views__ are everything the users can see and interact with. In DotVVM, the Views are written in __DOTHTML__, a flavour of _HTML_.
 
-A `@viewModel` __directive__ is at the beginning of every View. It tells DotVVM what ViewModel goes with it.
+You can see two enhancements to the plain HTML:
 
-A __Value Binding__ is a bond between the View and the ViewModel. It looks like this:
+* The line starting with `@viewModel` __directive__ tells DotVVM which ViewModel belongs to the page.
```

Comment:
Since we didn't mention directives before (neither of us), I'd like to rephrase that: "The line starting with `@viewModel` is the ViewModel Directive. It tells DotVVM which ViewModel belongs to the page."